src/js/api/posts/create.mjs

```diff
@@ -0,0 +1,26 @@
+import { authToken } from "../tokenFetch.mjs";
+
+/**
+ *
+ * @param {string} postUrl api url
+ * @param {string} method method used in api call
+ * @param {string} post data that is retrieved
+ * @returns {string} returns api data based on input (url, method, data)
+ */
+
+async function postData(postUrl, method, post) {
+  try {
+    console.log("postUrl", postUrl);
+    const response = await authToken(postUrl, {
+      method,
+      body: JSON.stringify(post),
+    });
+    const data = await response.json();
+
+    return data;
+  } catch (error) {
+    console.log(error);
```

Comment:
The user needs to know something went wrong, otherwise they will sit there wondering if their post succeeded

Reply:
I added more error checks visible for the user, it should be better now